src/game/models/asteroid.rs

```diff
@@ -144,6 +170,81 @@ impl Asteroid {
             on_screen: false,
         }
     }
+
+    pub fn can_split(&self) -> bool {
+        self.shape.len() > 10
+    }
+
+    pub fn split<P: Positioned>(&mut self, other: &P) -> Vec<Asteroid> {
+        self.normalize_rotation();
+        //let base_speed = self.vel();
```

Comment:
Ohhh, it showed the previous method as being highlighted on mobile, haha. Didn't see the comment. Yeah, I can take that out.